deploy/kubernetes/scenarios/wavefront-bring-your-own-logs-shipper.yaml

```diff
@@ -0,0 +1,22 @@
+# Need to change YOUR_CLUSTER_NAME, YOUR_WAVEFRONT_URL accordingly
+apiVersion: wavefront.com/v1alpha1
+kind: Wavefront
+metadata:
+  name: wavefront
+  namespace: observability-system
+spec:
+  clusterName: YOUR_CLUSTER_NAME
+  wavefrontUrl: YOUR_WAVEFRONT_URL
+  dataCollection:
+    metrics:
+      enable: true
+  dataExport:
+    # Once deployed, the svc is available to consume logs at
+    # For json_array format: http://wavefront-proxy.observability-system.svc.cluster.local:2878/logs/json_array?f=logs_json_arr
```

Comment:
I'd suggest not putting the the namespace for the proxy service url.  You can just put in `http://wavefront-proxy:2878/logs/json_array?f=logs_json_arr` instead and you don't have to worry about if they change the operator / proxy namespace.   That is how we previously configured fluentd.   See https://github.com/wavefrontHQ/wavefront-operator-for-kubernetes/blob/v2.1.0/deploy/internal/logging/0-fluent-configmap.yaml#L113

Reply:
Agree